engine/runtime/src/main/scala/org/enso/compiler/context/ChangesetBuilder.scala

```diff
@@ -42,8 +51,56 @@ final class ChangesetBuilder[A: TextEditor: IndexedSource](
     * @return the computed changeset
     */
   @throws[CompilerError]
-  def build(edits: Seq[TextEdit]): Changeset[A] =
-    Changeset(source, ir, compute(edits))
+  def build(edits: Seq[TextEdit]): Changeset[A] = {
+
+    def findSimpleUpdate(): SimpleUpdate = {
```

Comment:
why is this a nested local function? seems like it could perfectly be a private one outside of build

Reply:
It accesses `edits` variable from the closure.